src/KW_checkbox_internal.h

```diff
@@ -26,17 +26,19 @@
 #define KW_CHECKBOX_INTERNAL
 
 #include "KW_label.h"
+#include "KW_label_internal.h"
 #include "KW_widget.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
 typedef struct KW_Checkbox {
-  KW_Widget * label;
-  KW_bool     checked;
-  KW_Rect     selected;
-  KW_Rect     unselected;
+  /* so it can be used as a label... I think */
+  KW_Label label;
```

Comment:
Ideally widgets shouldn't use other widgets internal things, unless absolutely necessary. If there is a way of solving this without requiring label internal it would be better.